iocore/net/SSLClientUtils.cc

```diff
@@ -66,6 +66,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
   netvc = SSLNetVCAccess(ssl);
 
   if (netvc != nullptr) {
+    netvc->callHooks(TS_EVENT_SSL_SERVER_VERIFY_HOOK);
```

Comment:
BTW - What's the use case for this hook without an option to override OpenSSL validation - just for logging/auditing?